src/main/java/org/opendevstack/provision/services/openshift/OpenshiftService.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendevstack.provision.services.openshift;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.opendevstack.provision.adapter.exception.AdapterException;
+import org.opendevstack.provision.adapter.exception.CreateProjectPreconditionException;
+import org.opendevstack.provision.controller.CheckPreconditionFailure;
+import org.opendevstack.provision.model.OpenProjectData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.Assert;
+
+/** @author Sebastian Titakis */
```

Comment:
Probably just a nitpick but in my opinion Git is a better way to keep track of the actual author, so I basically agree with https://www.vojtechruzicka.com/stop-using-javadoc-author-tag/#what-is-wrong-with-author 
Is there a specific reason why you are using this annotation?

Reply:
Good point... I was following the convention introduced in ProvApp from the beginning... maybe we should remove all the noise someday.

Reply:
someday sounds like never, therefore I created a ticket 🤣  #633

Reply:
cool, thanks!